src/molecules/formfields/shared/formfields.module.scss

```diff
@@ -163,8 +167,31 @@ $checkbox-border-color: $neutral-5;
   }
 }
 
-// Select Field - types
 
+//
+.datefield {
+  composes: formfield;
```

Comment:
Since this file is getting so long, it might make sense break these all up into their own files -- if you do this, we would need to stop using `composes` and use a Sass placeholder instead to prevent class duplication.